packages/ffe-cards-react/package.json

```diff
@@ -6,10 +6,12 @@
   "author": "SpareBank 1",
   "files": [
     "lib",
-    "es"
+    "es",
+    "types"
   ],
   "main": "lib/index.js",
   "module": "es/index.js",
+  "types": "types/index.d.ts",
```

Comment:
Right, I just copied it into my node_modules folder for testing since npm linking has been broken since we started this monorepo stuff, missed that part.

Question: if all the "build" script does is to copy a file from `src/` to `types/` why not just put it in the `types/` directory in the first place?

Reply:
That's a good remark :+1:

Of course, we can put it there. I have no strong opinions on this, I think I just put it in src as all other written files were there, or I noticed a pattern for it somewhere. 

The only argument I see now is that the index.d.ts-files are in src/ in the other packages and it's a plus when the structure is the same across packages.